packages/core/src/components/Table/index.ts

```diff
@@ -14,5 +14,6 @@
  * limitations under the License.
  */
 
-export { default, TableColumn } from './Table';
+export { default } from './Table';
+export type { TableColumn } from './Table';
```

Comment:
`TableColumn` is actually defined as an `interface`. See https://github.com/spotify/backstage/blob/5363006759da372831a871f43162e8206d9351ac/packages/core/src/components/Table/Table.tsx#L150-L152

However doing
```
export interface { TableColumn } from './Table';
```
raises errors. `export interface` like this, is not supported.

Looking at the the `TableColumn` definition, I think it is okay to typecast `interface` to `type` and nothing is lost.

Reply:
The interface keyword is really just another way to define a type, so np